Source/WTF/wtf/posix/ThreadingPOSIX.cpp

```diff
@@ -609,7 +609,7 @@ bool ThreadCondition::timedWait(Mutex& mutex, WallTime absoluteTime)
     if (absoluteTime < WallTime::now())
         return false;
 
-    if (absoluteTime > WallTime::fromRawSeconds(std::numeric_limits<time_t>::max())) {
+    if (absoluteTime > WallTime::fromRawSeconds(static_cast<double>(std::numeric_limits<time_t>::max()))) {
```

Comment:
Hum yeah, this is not good indeed. Would it be better to do `>= (WallTime::fromRawSeconds(std::numeric_limits<time_t>::max()) + 1.0)`

Reply:
or `>= WallTime::fromRawSeconds(std::numeric_limits<time_t>::max() + 1.0)`

Reply:
Maybe the `>=` is the best we can do? `9223372036854775807` can't be represented in a double fp  :/

Reply:
yeah, i'm not sure... I just wanted to please the compiler... I'm sorry